### DISCUSSION TITLE
Integrating Sidekiq into a Hanami application

### DISCUSSION THREAD

**Discussion by tak1n** [hanamimastery/episodes#6] [Suggestions]
I would love to see an episode that leverages sidekiq for long-running operations. Some interesting topics to cover:

- Where would you place the worker class definitions?
- Does it make sense to have a new slice for sidekiq jobs?
- Is there some tighter integration planned? Like ActiveJob in rails world.

**Comment by swilgosz** [comment]
I will definitely explore this topic soon, especially that in Ascenda we use sidekiq heavily and it will be very useful to our existing applications upgrades. Thanks!

**Comment by swilgosz** [comment]
First part addressed in [HME027](https://hanamimastery.com/episodes/27-integrate-sidekiq-with-hanami) :). Thanks for the suggestion!
